Summarization.py

```diff
@@ -11,155 +11,168 @@
 import re
 import docx
 from nltk.tokenize import sent_tokenize
+import argparse 
+import sys
 #pip uninstall docx
 #pip install python-docx
-from docx import Document
-document = Document()
-document = Document('C:/Users/wb550776/Downloads/Using ML in Evaluative Synthesis May 31 2019.docx')
+#from docx import Document
+#document = Document()
+#document = Document('C:/Users/wb550776/Downloads/Using ML in Evaluative Synthesis May 31 2019.docx')
 
+
+def main():
+    parser= argparse.ArgumentParser()
+    parser.argument("/Users/minaekramnia/Documents/CV Cover Letter/Mina Ekramnia_CoverLetter_Educ.docx")

```

Comment:
this is the wrong syntax, and you don't want to pass a filename here, but the name of a variable
```
parse.add_argument('filename')
args = parser.parse_args()
summerize(args.filename)
```

```